source/install/install-rhel-66.rst

```diff
@@ -0,0 +1,23 @@
+.. _install-rhel-66:
+
+=================================
+Installing Mattermost on RHEL 6.6
+=================================
+
+Install a production-ready Mattermost system on 1 to 3 machines.
+
+.. include:: install-common-intro.rst
+
+.. contents:: Install and configure the components in the following order. Note that you need only one database, either MySQL or PostgreSQL.
+  :backlinks: top
+  :local:
+  
+.. include:: install-rhel-66-server.rst
```

Comment:
It's more neat and tidy to have a top-level file that includes the one-topic-per-file docs, even if some are (currently) single-use.